ci-operator/jobs/openshift/cincinnati/openshift-cincinnati-gh-pages-postsubmits.yml

```diff
@@ -0,0 +1,33 @@
+postsubmits:
+  openshift/cincinnati:
+  - agent: kubernetes
+    branches:
+    - gh-pages
```

Comment:
This is actually a regex... To avoid funky surprises, I suggest making it `^gh-pages$`